main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -1173,6 +1173,9 @@ public void setContent() {
 
             details.add(R.string.cache_type, cache.getType().getL10n());
             details.addSize(cache);
+            if (cache.getType() == CacheType.ADVLAB) {
```

Comment:
I would add the mode info above the size, to have it closer to the cache type (which I sort of regard the mode info being part of).

Reply:
ok ... will do

Reply:
Done